src/Plugin/resource/DataProvider/DataProviderEntity.php

```diff
@@ -231,19 +248,18 @@ public function view($identifier) {
     $id_field_name = empty($this->options['idField']) ? 'id' : $this->options['idField'];
     $resource_field_collection->setIdField($this->fieldDefinitions->get($id_field_name));
 
+    // Defer sparse fieldsets to the formatter. That way we can minimize cache
```

Comment:
I tried to explain https://github.com/RESTful-Drupal/restful/pull/660/files#r41708622

Does it make sense that way?